networking/network_security/network-policy-apis.adoc

```diff
@@ -17,7 +17,7 @@ Policies defined using the `AdminNetworkPolicy` API take precedence over all oth
 +
 Policies defined using the `BaselineAdminNetworkPolicy` API apply only when no other network policy overrides them. When you use the `AdminNetworkPolicy` API to delegate an aspect of network policy to the namespace-scoped `NetworkPolicy`, you should also define a sensible minimum restriction in the `BaselineAdminNetworkPolicy`. This ensures a baseline level of network security at the cluster level in case the `NetworkPolicy` for a namespace does not provide sufficient protection.
 
-Namespace-scoped network policy:: Application developers and namespace tenants can use the `NetworkPolicy` API to define network policy rules for a specific namespace. Rules in the `NetworkPolicy` for a namespace take precedence over cluster-wide rules configured using the BaselineAdminNetworkPolicy API, or for a cluster-wide rule that that has been delegated or "passed" from the cluster-wide `AdminNetworkPolicy` API. 
+Namespace-scoped network policy:: Application developers and namespace tenants can use the `NetworkPolicy` API to define network policy rules for a specific namespace. Rules in the `NetworkPolicy` for a namespace take precedence over cluster-wide rules configured using the BaselineAdminNetworkPolicy API, or for a cluster-wide rule that has been delegated or "passed" from the cluster-wide `AdminNetworkPolicy` API. 
 
 [discrete]
```

Comment:
@bergerhoffer do you mean here? it's not a part of the git diff, so the CI will not check against it

Reply:
i'm uncertain if changing things so that all alerts are reflected in PR comments is possible, based on the GH API the prow script uses: there's a comment that says the GH API only accepts comments within the diff for the review comments endpoint

Reply:
Yeah I'm not saying we need to change it. I just want to make sure I understand exactly what it's doing so that we can make sure everyone else is aware as well. For some reason, I thought I had made a change to a file elsewhere and it picked something up. I'll check back through my test PRs to see if I can find what I'm talking about. Thanks!